### DISCUSSION TITLE
WPF developers looking for a good one

### DISCUSSION THREAD

**Discussion by ChenYuan516** [dotnet/wpf#5587] [Show and tell]
Find a good one https://github.com/yanjinhuagood/WPFDevelopers

**Comment by lindexi** [comment]
Awesome!

**Comment by yanjinhuagood** [comment]
![RadarCha](https://user-images.githubusercontent.com/23089734/139358324-079012cb-56ec-4f3c-af32-c1edb0c6ea2a.gif)
![SpotLight](https://user-images.githubusercontent.com/23089734/139358369-2eb0ef63-b953-4c4f-a431-05913c1114a8.gif)
![BasicBarChart](https://user-images.githubusercontent.com/23089734/139582341-8fcf8580-2161-45e3-8cd8-eb2d87fa01b4.gif)
![BingAMap](https://user-images.githubusercontent.com/23089734/139582367-cfcd8971-9234-4394-afb1-ad032f2897d4.gif)
![CircularMenu](https://user-images.githubusercontent.com/23089734/139582378-5dac352d-26a3-4e5d-8f42-2d9f077b8915.gif)
![TransitionPanel](https://user-images.githubusercontent.com/23089734/139582382-2933a292-00db-4528-a7f1-059f9bb177ab.gif)
![ChatEmojiText](https://user-images.githubusercontent.com/23089734/139582389-4355137c-d5c1-4e6a-bb60-6903823eec5e.gif)

**Reply by jamesnet214** [reply]
I am a WPF developer from South Korea. A while ago, I introduced and shared this [repository](https://github.com/yanjinhuagood/WPFDevelopers) in a Korean community. This repository, like a gift set of various open-source projects, has provided great inspiration to Korean developers. It also served as an opportunity to think about the importance of open-source.

https://forum.dotnetdev.kr/t/opensource/8348/4?u=jamesnet214

**Reply by yanjinhuagood** [reply]
Thank you for sharing, big shot
